openshift-gitops-operator/templates/argocd.yml.j2

```diff
@@ -0,0 +1,71 @@
+          metadata:
+            labels:
+              opdev: opcap
+          spec:
+            server:
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+                requests:
+                  cpu: 125m
+                  memory: 128Mi
+              route:
+                enabled: true
+            repo:
+              resources:
+                limits:
+                  cpu: 1000m
+                  memory: 1024Mi
+                requests:
+                  cpu: 250m
+                  memory: 256Mi
+            sso:
+              provider: dex
+              dex:
+                openShiftOAuth: true
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 256Mi
+                  requests:
+                    cpu: 250m
+                    memory: 128Mi
+            ha:
+              enabled: false
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+                requests:
+                  cpu: 250m
+                  memory: 128Mi
+            rbac:
+              defaultPolicy: ''
+              policy: |
+                g, system:cluster-admins, role:admin
+              scopes: '[groups]'
+            redis:
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 256Mi
+                requests:
+                  cpu: 250m
+                  memory: 128Mi
+            resourceExclusions: |
+              - apiGroups:
+                - tekton.dev
+                clusters:
+                - '*'
+                kinds:
+                - TaskRun
+                - PipelineRun
+            controller:
+              resources:
+                limits:
+                  cpu: 2000m
+                  memory: 2048Mi
+                requests:
+                  cpu: 250m
+                  memory: 1024Mi
```

Comment:
Needs a newline.

Reply:
Hi @bcrochet , I added the newline but just let it noted ansible-lint is not picking this up. Not sure if for Jinja files this is actually needed.